core/main.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
```

Comment:
Can we move this to `providers.tf` same in `shared` too.

Reply:
Updated all in [2ba81bf](https://github.com/nvminhtue/tyson_devops_ic/pull/18/commits/2ba81bf83d844be10e367ffa8ef3f3454451b706)